### PR TITLE
Task 2.1 Neural Network

### PR DESCRIPTION
**Pull request by 01gaM**
Neural net test set accuracy: 0.182000
Ищу и исправляю ошибки

### CODE REVIEW

**Comment by 01gaM** [comment]
@toshiks 
Neural net test set accuracy: 0.682000 
Были подобраны другие параметры для тренировки нейросети, точность значительно повысилась
Лучшее значение val accuracy:
Loss: 0.847375, Train accuracy: 0.777778, val accuracy: 0.734000

**Comment by 01gaM** [comment]
@toshiks 
Задание 2.2
С использованием PyTorch было натренировано несколько моделей
Для последней модели точность на тестовой выборке (Test accuracy): 0.8075
